src/polymer/create-element-extension.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import babelTraverse from 'babel-traverse';
+import * as babel from 'babel-types';
+import {Document, Severity, Warning} from 'polymer-analyzer';
+
+import {registry} from '../registry';
+import {Rule} from '../rule';
+import {getDocumentContaining, stripIndentation} from '../util';
+
+
+class CreateElementExtension extends Rule {
+  code = 'create-element-extension';
+  description = stripIndentation(`
+    Warns when using the second parameter of createElement for element extension using the is attribute.
+  `);
+
+  async check(document: Document) {
+    const warnings: Warning[] = [];
+
+    const docs = document.getFeatures({kind: 'js-document'});
+
+    for (const doc of docs) {
+      babelTraverse(doc.parsedDocument.ast, {
+        noScope: true,
+        CallExpression: (path) => {
+          if (!this.isExtendingElementCall(path.node)) {
+            return;
+          }
+
+          let message: string;
+          if (babel.isStringLiteral(path.node.arguments[1])) {
```

Comment:
@rictic Let me know what you think about this? I don't know quite how you would want to handle different messages for string vs object or should I just use a single message?

Reply:
I'm also curious if I should only highlight the second param or the way I have with the entire call expression?

Reply:
I think a separate message and a separate code is called for here. Some teams may want to ignore one warning but not the other.

Reply:
Highlighting the second param is more specific and helpful I think.